src/main/java/ch/uzh/ifi/fs22/sel/group15/cozycave/server/service/ListingService.java

```diff
@@ -47,6 +46,10 @@ public List<Listing> getListings() {
         return this.listingRepository.findAll();
     }
 
+    public List<Listing> getListingsFiltered() {
```

Comment:
not using `getListings()` for better changing later?

Reply:
I think you misunderstood me (and as I read my comment again, its very misleading): the current version doesn't filter anything. because of that, I wouldn't name it `getListingsFiltered()`.

Before, there where two individual methods both doing the same but named differently. I just wanted to point that out.

So I think you have to change the name back :D